trio_model.cc

```diff
@@ -408,15 +411,29 @@ void TrioModel::SequencingProbabilityMat() {
 void TrioModel::SomaticTransition(bool is_numerator) {
   if (!is_numerator) {
     read_dependent_data_.denominator.child_zygotic_probability = (
-      read_dependent_data_.child_somatic_probability * somatic_probability_mat_
+            read_dependent_data_.child_somatic_probability * somatic_probability_mat_
     );
     read_dependent_data_.denominator.mother_zygotic_probability = (
-      read_dependent_data_.mother_somatic_probability * somatic_probability_mat_
+            read_dependent_data_.mother_somatic_probability * somatic_probability_mat_
     );
     read_dependent_data_.denominator.father_zygotic_probability = (
-      read_dependent_data_.father_somatic_probability * somatic_probability_mat_
+            read_dependent_data_.father_somatic_probability * somatic_probability_mat_
     );
 
+//    Note: example with unordered_map<key, zygotic_probability> and encap encapsulation
+//     unordered_map<string, RowVector16d> read_dependent_data_.denominator.zygotic
+//
+//    CalculateSomaticProbability(read_dependent_data_.denominator, somatic_probability_mat_)
+//    CalculateSomaticProbability(read_dependent_data_.numerator, somatic_probability_mat_diag_)
+
+//    void CalculateSomaticProbability(TreePeel &tree_peel, double scale){
+//      for( auto key : tree_peel.zygotic){
+//        tree_peel.somatic[key] = tree_peel.zygotic[key] * somatic_probability_mat_;
+//        //Even better, assign scale to TreePeel:    tree_peel.somatic[key] = tree_peel.zygotic[key] * tree_peel.scale
```

Comment:
I don't understand what scale is for. Also, it would be zygotic = somatic * somatic probability mat because I have meant here SomaticTransition is the transition using somatic probabilities to zygotic.